apps/list.c

```diff
@@ -1343,8 +1343,6 @@ static void list_disabled(void)
 #endif
 #ifdef OPENSSL_NO_EC
     BIO_puts(bio_out, "EC\n");
-#endif
-#ifdef OPENSSL_NO_ECX
     BIO_puts(bio_out, "ECX\n");
```

Comment:
ECX is no longer a thing, so this should be removed

Reply:
will fix.